src/matchers/toBeArray/predicate.test.js

```diff
@@ -0,0 +1,12 @@
+import each from 'jest-each';
+import predicate from './predicate';
+
+describe('toBeArray Predicate', () => {
+  it('returns true when given a string', () => {
```

Comment:
I think the test description is meant to be: 

`returns true when given an array`